pkg/cluster/storegaaccounts.go

```diff
@@ -0,0 +1,89 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"strings"
+
+	mgmtnetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-07-01/network"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+
+	"github.com/Azure/ARO-RP/pkg/util/arm"
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+)
+
+// enableServiceEndpointstEndpoints should enable Microsoft.Storage endpoints on
+// subnets for storage account access
+func (m *manager) enableServiceEndpointstEndpoints(ctx context.Context) error {
```

Comment:
I don't like this function name because it looks like a typo even though it probably isn't

Reply:
it is :) good catch